include/enum_quda.h

```diff
@@ -129,11 +129,26 @@ extern "C" {
   } QudaInverterType;
 
   typedef enum QudaEigType_s {
-    QUDA_LANCZOS, //Normal Lanczos eigen solver
-    QUDA_IMP_RST_LANCZOS, //implicit restarted lanczos solver
-    QUDA_INVALID_TYPE = QUDA_INVALID_ENUM
+    QUDA_LANCZOS,           // Lanczos solver
```

Comment:
This is probably a good chance to clean up the enums here.  Perhaps `QUDA_EIG_LANCZOS`, QUDA_EIG_ARNOLDI`, etc. here.

Also, I'd prefer `QUDA_SPECTRUM_SR_EIG` over `QUDA_SR_EIG_SPRECTRUM`.  Not all enums follow this convention of increasing specificity as we move from left to right, but that was a poor early design choice.  Since this is a new type, let's get this one correct 😉